satellite/marketingweb/server.go

```diff
@@ -135,7 +141,7 @@ func (s *Server) parseTemplates() (err error) {
 
 	s.templates.home, err = template.New("home-page").Funcs(template.FuncMap{
 		"BaseURL":      s.GetBaseURL,
-		"ReferralLink": rewards.GeneratePartnerLink,
+		"ReferralLink": s.partners.GeneratePartnerLink,
```

Comment:
Why not pass in `s.generatePartnerLink` ?

Reply:
Ah, yes, that's a mistake.

Reply:
should be fixednow